### PR TITLE
Operator base image change to scratch.

### PR DESCRIPTION
**Pull request by libesz**
Since we do not install or use any commands at production, let's not be involved with any vulnerabilities of the operating systems, and use scratch images.

### CODE REVIEW

**Comment by mihivagyok** [comment]
One other issue is that `make lint-sh` is failing.

**Comment by libesz** [comment]
🎉

File: scripts/fvt-tools.sh [hunk 1]
```diff
@@ -9,23 +9,50 @@ fvtlog() {
...
+delete_hostnet_pods() {
+  kubectl delete po --selector fvt-helper=hostnet
+}
+
+# Function to execute a command on the host network of a node, selected by a pod that is running on it
+# Parameters:
+# - Node name
+# - Command (may be multiple string)
+exec_in_hostnet_of_node() {
+  nodename=$1
```
Comment (mihivagyok) [inline scripts/fvt-tools.sh:48]: Could you please add the word `local` to this variable?

Comment (libesz) [reply scripts/fvt-tools.sh]: Yep

**Pull request merged**